Actions/IncrementVersionNumber/IncrementVersionNumber.psm1

```diff
@@ -0,0 +1,171 @@
+﻿<#
+    .Synopsis
+        Changes a version setting value in a settings file.
+    .Parameter settingsFilePath
+        Path to a JSON file containing the settings.
+    .Parameter settingName
+        Name of the setting to change. The setting must be a version number.
+    .Parameter newValue
+        New value of the setting. Allowed values are: +1 (increment major version number), +0.1 (increment minor version number), or a version number in the format Major.Minor (e.g. 1.0 or 1.2
+#>
+function Set-VersionInSettingsFile {
```

Comment:
Minor: Name implies that it always sets the version - but it only sets it if it exists already

Reply:
Actually - thinking about this - we need to update the repoversion even if it isn't there.
Else the next version will just be 1.0 as well - the repoversion defaults to 1.0 and is used as filename for the artifacts for a project

Reply:
Ok, added functionality to ensure `repoVersion` is set in the repo settings.

For project settings, `repoVersion` will be changed only if it existed before.